vscode-client/extension.ts

```diff
@@ -117,6 +116,10 @@ export async function activateAsync(context: vscode.ExtensionContext): Promise<v
     new KeywordsTreeViewProvider(context, languageClientManger, outputChannel),
   );
 
+  if (vscode.workspace.getConfiguration("robotcode.testView").get<boolean>("enabled")) {
```

Comment:
This is not the correct place to disable the test explorer support. 

1. this code here is only called once when the extension is started, so you have to reload the entire extension or Vscode if you activate/deactivate this setting.
2. you can make settings at user/workspace and folder level. What happens if I open two projects at the same time and want to use the normal RobotCode tester ID in the first project, while I want your Pytest support in the second project? This code does not support this.

Reply:
i had another go at this. does what i have now look like a better approach? sorry if it's still shit, i'm quite new to vscode extension development and i still can't get it to build so i don't even know if it works lol